### ISSUE TITLE
Can MaryTTS run without server ? 

### ISSUE THREAD

**Issue by Kawrantin** [marytts/marytts#518]
Hi, 

Excuse me for my poor English :)
I have a project as follow :  Use MaryTTS on Windows CE 6.0

But a problem occured, I use the #119 's final code : 

```java
String text = "Hello World";
MaryInterface marytts = new LocalMaryInterface();
marytts.setInputType("RAWMARYXML");
Set<String> voices = marytts.getAvailableVoices();
marytts.setVoice(voices.iterator().next());
Document document = MaryXML.newDocument();
Element maryxml = document.getDocumentElement();
maryxml.setAttribute("xml:lang", "en-US");
Element paragraph = MaryXML.appendChildElement(maryxml, MaryXML.PARAGRAPH);
Element sentence = MaryXML.appendChildElement(paragraph, MaryXML.SENTENCE);
Element t = MaryXML.appendChildElement(sentence, MaryXML.TOKEN);
t.setTextContent(text);
MaryData maryData = new MaryData(MaryDataType.PHONEMES, Locale.ENGLISH, false);
maryData.setDocument(document);
AudioInputStream audio = marytts.generateAudio(document);
AudioPlayer player = new AudioPlayer(audio);
player.start();
player.join();
```

And I get this error : 
```
Exception in thread "main" java.util.NoSuchElementException
	at java.util.HashMap$HashIterator.nextNode(Unknown Source)
	at java.util.HashMap$KeyIterator.next(Unknown Source)
	at texte.Chiffre.main(Chiffre.java:264) 

line 264 : marytts.setVoice(voices.iterator().next());
```

I have an other question. As I said, I work on Windows CE, so I can't run a server and a Java program at the same time. (low memory)
My question is : Is it possible to duplicate the server's actions and behaviour in the main, using the marytts/marytts-runtime/src/main/java/marytts/server/'s source code ? 
In other words, make MaryTTS speak withour runnig the server ?

**Comment by seblemaguer** [comment]
HEllo,

to the main question the answer is yes. However your solution is not the good one. You have an example here about how to do that => https://raw.githubusercontent.com/marytts/marytts-txt2wav/gradle/src/main/java/de/dfki/mary/Txt2Wav.java
